package.json

```diff
@@ -75,6 +75,6 @@
     "backbone.paginator": "~2.0.3",
     "moment": "^2.15.1",
     "moment-timezone": "~0.5.5",
-    "sinon": "1.17.3 || >1.17.4"
+    "sinon": "1.17.3 || 1.17.7"
```

Comment:
now that we're not using NPM < 3 anywhere, I don't think we actually need peerDependencies at all?